xml/decoders/Atlas_Lenz_N.xml

```diff
@@ -12,12 +12,14 @@
 <!-- FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License  -->
 <!-- for more details.                                                      -->
 <decoder-config xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/decoder-4-15-2.xsd">
+  <version author="Peter Ulvestad" version="3" lastUpdated="20240612"/>
   <version author="Alain Le Marchand" version="2" lastUpdated="20180216"/>
   <version author="Alain Le Marchand" version="1.1" lastUpdated="20140309"/>
   <version author="Alain Le Marchand" version="1" lastUpdated="20140308"/>
   <!-- Version 1 - Created from Lenz_LE077XF.xml v7.1 - Atlas branded versions of LE062XF and LE063FX (Alain Le Marchand) -->
   <!-- Version 1.1 - Change description for light options to match reality - unclear manual from Atlas - Set default Manufacturer address to 127 = Atlas (Alain Le Marchand) -->
   <!-- Version 2 - Implement Reset properly (Alain Le Marchand) -->
+  <!-- Version 3 - Added MP15DC (Peter Ulvestad)
```

Comment:
This line is missing the end of comment `-->`

Reply:
Opps, fixed it